command/v7/map_route_command.go

```diff
@@ -106,7 +121,7 @@ func (cmd MapRouteCommand) Execute(args []string) error {
 		cmd.UI.DisplayOK()
```

Comment:
Should we include some extra error here if the protocol is set and doesn't match the retrieved route destination's protocol? Otherwise the user may believe that they've set the protocol successfully.

Reply:
I agree, we should add an extra statement here to indicate that nothing has changed.